accounts/abi/bind/bind.go

```diff
@@ -119,14 +119,11 @@ func Bind(types []string, abis []string, bytecodes []string, fsigs []map[string]
 			normalized.Inputs = make([]abi.Argument, len(original.Inputs))
 			copy(normalized.Inputs, original.Inputs)
 			for j, input := range normalized.Inputs {
-				// Indexed fields are input, non-indexed ones are outputs
-				if input.Indexed {
```

Comment:
Because I think the original comment is not correct. 
In this step, we only assign the name for anonymous parameters of the event and try to extract structure definition from parameters.

Regarding the "Indexed" and "non-indexed" parameters, we don't need to distinguish here. Since during the log unpacking, "Indexed" and "non-indexed" parameters are treated differently.